lms/services/digest.py

```diff
@@ -89,7 +88,7 @@ def send_instructor_email_digests(  # pylint:disable=too-many-arguments
                 recipient=asdict(EmailRecipient(to_email, user_info.display_name)),
                 template_vars=digest,
                 unsubscribe_url=self._email_preferences_service.unsubscribe_url(
-                    user_info.h_userid, EmailUnsubscribe.Tag.INSTRUCTOR_DIGEST
+                    user_info.h_userid, "instructor_digest"
```

Comment:
We're still including these tags in the unsubscribe links although they're no longer actually used for anything. But if we ever have multiple types of unsubscribe links we'll be able to distinguish them based on the tags encoded in the links.